nix/nix/sources.json

```diff
@@ -36,9 +36,9 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "tezos": {
-        "ref": "v7.3",
+        "ref": "v7.4",
```

Comment:
This is going to break with nix 3.0, but I've got a fix almost ready (it takes all the tezos packages from OPAM instead of the custom repo)